apps/server/src/modules/rocketchat/rocket-chat.module.ts

```diff
@@ -7,7 +7,11 @@ export class RocketChatModule {
 	static forRoot(options: RocketChatOptions): DynamicModule {
 		return {
 			module: RocketChatModule,
-			imports: [HttpModule],
+			imports: [
+				HttpModule.register({
+					timeout: 30000,
```

Comment:
Please put it in some environment variable, e.g. `ROCKET_CHAT_CLIENT_TIMEOUT_MS` with default value `30000`.

Reply:
done